pom.xml

```diff
@@ -86,7 +86,7 @@
     <version.maven-compiler-plugin>3.8.1</version.maven-compiler-plugin>
     <version.maven-enforcer-plugin>1.4.1</version.maven-enforcer-plugin>
     <version.maven-failsafe-plugin>3.0.0-M5</version.maven-failsafe-plugin>
-    <version.maven-gpg-plugin>1.6</version.maven-gpg-plugin>
+    <version.maven-gpg-plugin>3.0.1</version.maven-gpg-plugin>
```

Comment:
Why is this change here?

Reply:
@rohanKanojia it automatically made that change. I didn't edit this file at all!

Reply:
Could you please revert this change and keep changes limited to issue scope?